### ISSUE TITLE
Frontend: Display for password forgotten display

### ISSUE THREAD

**Issue by BibaltiK** [BibaltiK/Hackathon#30]
Currently, there is no feedback when a new password is requested, which is obviously not very user-friendly.

**Comment by realArafatBen** [comment]
Do you mean no email request was sent?

**Comment by BibaltiK** [comment]
No, the email is sent. Only the feedback to the user is missing.
Is only a small thing, but was late yesterday ;)

**Comment by realArafatBen** [comment]
Okay. 
I will take a look. Right now

**Comment by BibaltiK** [comment]
It requires any change in the template after the response in the CreateNewPasswordView

**Comment by BibaltiK** [comment]
Hello,
are you still there? Otherwise I would do it quickly now

**Comment by realArafatBen** [comment]
Yeah.. 
Am working on it but am getting 404 error on the apis when I run npm run serve

**Comment by BibaltiK** [comment]
Oh yes, with `npm run serve` it doesn`t work either, because it is not a classic API client application.
The delivery from the frontend is done by the first call from the backend.
The easiest way to do this is via a Docker container via ddev.
I have adjusted the project and the README.MD again, because I noticed two more errors. So please get the project again and import the live.sql.
After own test it works actually smoothly.
I should once again deposit how the project is started.
In short, the first call must be via index.php via Apache or nginx.

**Comment by realArafatBen** [comment]
Okay. I will get back to u.

**Comment by realArafatBen** [comment]
Hi, 
This is my first time using Docker... 
am using Apache.. 
How do I set my env for this 
thanks

**Comment by BibaltiK** [comment]
Hey,

you can also just use your existing Apache environment. Makes more sense in this case.

But to the question, since the Docker configuration is more complex, I personally use the tool ddev for this.
